### PR TITLE
feature(tester): added test result event

### PR DESCRIPTION
**Pull request by bentsi**
Currently if the test failed by assert we won't
receive correct status in email report since it
is based on checking critical events

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/tester.py [hunk 1]
```diff
@@ -1629,18 +1627,8 @@ def list2reason(exc_list):
         else:  # Python 3.2 - 3.3 or 3.0 - 3.1 and 2.7
             result = getattr(self, '_outcomeForDoCleanups', self._resultForDoCleanups)  # pylint: disable=no-member
         error = list2reason(result.errors)
-        failure = list2reason(result.failures)
-        ok = not error and not failure
-
-        if not ok:
-            typ, text = ('ERROR', error) if error else ('FAIL', failure)
-            TEST_LOG.error("=" * 70)
```
Comment (fruch) [inline sdcm/tester.py]: why did you remove this prints ?  since the event is printed ?

**Comment by fruch** [comment]
@bentsi don't we want to backport it to 3.3 ?

File: sdcm/tester.py (hunk 1 above)
Comment (bentsi) [reply sdcm/tester.py]: yes, I print the event

**Comment by bentsi** [comment]
@fruch we can. Let's see how it works in master
